renku/core/errors.py

```diff
@@ -212,28 +211,16 @@ class InvalidOutputPath(RenkuException):
 class OutputsNotFound(RenkuException):
     """Raise when there are not any detected outputs in the repository."""
 
-    def __init__(self, repository, inputs):
+    def __init__(self):
         """Build a custom message."""
-        from pathlib import Path
-
-        msg = "There are not any detected outputs in the repository."
-
-        paths = [
-            os.path.relpath(input_.default_value)  # relative to cur path
-            for input_ in inputs
-            if Path(input_.default_value).is_dir()
-        ]
-
-        if paths:
-            msg += (
-                '\n  (use "git rm <file>..." to remove them first)'
-                "\n\n" + "\n".join("\t" + click.style(path, fg="yellow") for path in paths) + "\n\n"
-                "Once you have removed files that should be used as outputs,\n"
-                "you can safely rerun the previous command."
-                "\nYou can use --output flag to specify outputs explicitly."
-            )
-        else:
-            msg += "\n\nIf you want to track the command anyway use " "--no-output option."
+
+        msg = (
+            "There are not any detected outputs in the repository. This can be due to your command not creating "
+            "any new files or due to files that get created already existing before the command was run. In the "
```

Comment:
I'm not sure about this: `already existing` -> `already exist`

Reply:
I think plural is correct, `due to files [...] already existing`